--- update.sh	2025-10-09 06:15:53.886809000 +0000
+++ update.sh.fuzz	2025-10-09 06:17:58.186621000 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e
 
-openbsd_branch=master
+openbsd_branch=ff933db9f
 if [ -n "$1" ]; then
 	openbsd_branch="$1"
 fi
@@ -19,8 +19,7 @@
 if [ -d openbsd/.git ]; then
 	(cd openbsd
 	 git fetch
-	 git checkout "${openbsd_branch}"
-	 git pull --rebase)
+	 git checkout "${openbsd_branch}")
 fi
 
 # setup source paths
